extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -223,14 +227,31 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
   }
 
   /**
-   * Force this extension to behave as if ad is about to be shown.
-   * Used for testing but should be extended to force an ad to show in a real story.
-   * @param {string} pageBeforeAdId
+   * Force this extension to render all amp-story-auto-ads controlled UI and
```

Comment:
Force an immediate ad placement without waiting for ad being loaded, and then navigate to the ad page.

Reply:
Done.